locales/de.json

```diff
@@ -0,0 +1,51 @@
+{
+    "settings": {
+        "title": "Einstellungen",
+        "legend": "URI zur .ics-Datei",
+        "calendar_name": "Kalendername",
+        "calendar_uri": "URI",
+        "new_item_button": "Neuer Kalender",
+        "save_button": "Speichern",
+        "remove_button": "Löschen",
+        "saved_alert": "Einstellungen gespeichert",
+        "uri_load_failure": "Das Laden der .ics-Datei(en) ist fehlgeschlagen:",
+        "icons_info": "Icons von"
+    },
+    "conditions_event_name_start_date_time_format": "MM.DD HH:mm",
+    "conditions_event_name_start_date_year_time_format": "MM.DD.YY HH:mm",
+    "conditions_event_name_stop_time_format": "HH:mm",
+    "conditions_event_name_stop_date_time_format": "MM.DD HH:mm",
+    "conditions_event_name_stop_date_year_time_format": "MM.DD.YY HH:mm",
+    "conditions_event_name_start_stop_date_format": "MM.DD",
+    "conditions_event_name_start_stop_date_year_format": "MM.DD.YY",
+    "token_duration_minutes": "Minuten",
+    "token_duration_hour": "Stunde",
+    "token_duration_hours": "Stunden",
+    "calendarTokens": {
+        "events_today_calendar_title_stamps": "Heutige Ereignisse in",
+        "events_tomorrow_calendar_title_stamps": "Morgige Ereignisse in"
+    },
+    "flowTokens": {
+        "event_next_title": "Nächster Veranstaltungstitel",
+        "event_next_startstamp": "Beginn der nächsten Veranstaltung",
+        "event_next_startstamp_date_time_format": "MM.DD HH:mm",
+        "event_next_startstamp_date_format": "MM.DD",
+        "event_next_stopstamp": "Ende der nächsten Veranstaltung",
+        "event_next_stopstamp_date_time_format": "MM.DD HH:mm",
+        "event_next_stopstamp_date_format": "MM.DD",
+        "event_next_duration": "Nächste Veranstaltungsdauer",
+        "event_next_duration_minutes": "Nächste Veranstaltungsdauer (Minuten)",
+        "event_next_starts_in_minutes": "Nächste Veranstaltung beginnt in (Minten)",
+        "event_next_stops_in_minutes": "Nächste Veranstaltung endet in (Minuten)",

```

Comment:
Please change in: "Nächstes Ereignis endet in (Minuten)"

Reply:
Done